icons/bbciplayer.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>BBC iPlayer</title><path d="M8.315 0H2.382v6.022h5.933V3.506l9.618 8.45-9.618 8.538V8.99H2.382V24h5.933l13.303-12.045Z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>BBC iPlayer</title><path d="M1.741 4.789H6.52c.085 0 .154.069.154.154v14.114a.154.154 0 0 1-.154.155H1.741a.154.154 0 0 1-.154-.155V4.943c0-.085.069-.154.154-.154ZM7.513 4.217l2.39-4.14a.153.153 0 0 1 .21-.056l12.224 7.058a.153.153 0 0 1 .056.21l-2.39 4.139a.153.153 0 0 1-.21.056L7.57 4.427a.154.154 0 0 1-.057-.21ZM7.513 19.784l2.39 4.139a.154.154 0 0 0 .21.057l12.224-7.058a.154.154 0 0 0 .056-.21l-2.39-4.14a.154.154 0 0 0-.21-.056L7.57 19.574a.154.154 0 0 0-.057.21Z"/></svg>
```

Comment:
I'm seeing a slight difference on this iPlayer logo from the SVG embedded in the header (overlaid in colour onto yours in black) happening in both diagonal arms of the logo...

![image](https://github.com/simple-icons/simple-icons/assets/1830867/dfbf9e60-bd19-4454-90ac-1a9a40a31da1)

Might be easily fixable by either re-rendering using that source, or upping the decimals when using SVGO.

Reply:
I can try updating these again, but was getting validation errors that I couldn’t get around without these adjustments. Will update again and see if those errors come back, and list them here.

Reply:
Yeah, if you run SVGO through [SVGOMG](https://jakearchibald.github.io/svgomg/) instead of the command line, you might need to set the precision decimals to 4. Sometimes these icons can just be like that 🙃

Reply:
I get the following error using the SVG from the BBC website:

```
icon-centered <path> must be centered at (12, 12); the center is currently (12.002, 12)
```

I get this with a precision of 4 or 5; with 3 I get more errors. It’s hard to know how to fix this as I guess the centre is getting calculated based on where the slight curve of the point is being rendered.

Reply:
I'll take a quick look at the iPlayer one myself today, see if I get any issues. If not, I'll share my SVG path here so hopefully it'll work on your end too :)